runtime/client_http_request.go

```diff
@@ -72,16 +75,22 @@ func NewClientHTTPRequest(
 
 	ctx = WithScopeTags(ctx, scopeTags)
 	req := &ClientHTTPRequest{
-		ClientID:             clientID,
-		MethodName:           clientMethod,
-		ClientTargetEndpoint: clientTargetEndpoint,
-		Metrics:              client.contextMetrics,
-		client:               client,
-		ContextLogger:        client.ContextLogger,
-		defaultHeaders:       client.DefaultHeaders,
-		ctx:                  ctx,
-		jsonWrapper:          client.JSONWrapper,
+		ClientID:               clientID,
+		MethodName:             clientMethod,
+		ClientTargetEndpoint:   clientTargetEndpoint,
+		Metrics:                client.contextMetrics,
+		client:                 client,
+		ContextLogger:          client.ContextLogger,
+		defaultHeaders:         client.DefaultHeaders,
+		ctx:                    ctx,
+		jsonWrapper:            client.JSONWrapper,
+		timeoutAndRetryOptions: timeoutAndRetryOptions,
 	}
+	req.ContextLogger.Debug(req.ctx, "ClientHTTPRequest definition",
+		zap.String("clientId", req.ClientID), zap.String("methodName", req.MethodName),
+		zap.String("req.RequestTimeoutPerAttemptInMs", string(rune(req.timeoutAndRetryOptions.RequestTimeoutPerAttemptInMs))),
```

Comment:
why not use `zap.Int` instead to avoid the type conversion?

Reply:
Make sense, Addressed

Reply:
RequestTimeoutPerAttemptInMs is in time.Duration format - hence I kept it as string